examples/memory.rs

```diff
@@ -0,0 +1,89 @@
+use wasmer::{
+    imports, wat2wasm, Extern, Function, Instance, Memory, MemoryType, Module, NativeFunc, Pages,
+    Store, Table, TableType, Type, Value,
+};
+use wasmer_compiler_cranelift::Cranelift;
+use wasmer_engine_jit::JIT;
+
+// this example is a work in progress:
+// TODO: clean it up and comment it
+
+fn main() -> anyhow::Result<()> {
+    let wasm_bytes = wat2wasm(
+        r#"
+(module
+  (type $mem_size_t (func (result i32)))
+  (type $get_at_t (func (param i32) (result i32)))
+  (type $set_at_t (func (param i32) (param i32)))
+
+  (memory $mem 1)
+  ;;(import "env" "memory" (memory $mem 1))
```

Comment:
?

Reply:
Oh this was testing code for testing the control flow of imported vs local/exported memory